test/Test/Data/Main.purs

```diff
@@ -0,0 +1,149 @@
+module Test.Data.GenericsRep where
```

Comment:
Should we name this module Test.Data.Enum.Generic, since we’re testing the generic implementations exported from Data.Enum.Generic?

Reply:
Fixed